gradle/libs.versions.toml

```
@@ -0,0 +1,15 @@
+[versions]
+android-gradlePlugin = "8.1.4"
+android-sdkBuildTools = "31.1.1"
+spock = "2.3-groovy-3.0"
+guava = "33.4.0-jre"
+
+[libraries]
+android-tools-gradlePlugin = { module = "com.android.tools.build:gradle", version.ref = "android-gradlePlugin" }
+android-tools-common = { module = "com.android.tools:common", version.ref = "android-sdkBuildTools" }
+android-tools-sdkCommon = { module = "com.android.tools:sdk-common", version.ref = "android-sdkBuildTools" }
```

What do you think about this?

```suggestion
android-gradlePlugin = { module = "com.android.tools.build:gradle", version.ref = "android-gradlePlugin" }
android-common = { module = "com.android.tools:common", version.ref = "android-sdkBuildTools" }
android-sdkCommon = { module = "com.android.tools:sdk-common", version.ref = "android-sdkBuildTools" }
```

---

That's better. https://github.com/gradle/android-cache-fix-gradle-plugin/pull/1754/commits/ddc42aab7ae4fdc63b43f4e09184110435f58831